setup.py

```diff
@@ -47,7 +47,7 @@
 ]
 
 install_requires = [
-    'mosaicml[libcloud,wandb,mlflow]>=0.15.0,<0.16',
+    'mosaicml[libcloud,wandb,mlflow]>=0.16,<0.17',
```

Comment:
@dakinggg Is it okay to upgrade the mosaicml version? We need at least 0.16 for ObjectStore.list_objects